.github/CONTRIBUTING.md

```diff
@@ -30,9 +30,21 @@ If you write and contribute a full topic, we will add your name (or your company
 1. Create a new branch on your fork. Use a name that best describes the work, or references a GitHub issue number.
 1. Edit or create markdown (`.md`) files in your branch.
 1. When ready, push your branch to your fork.
-1. Create a PR to the [magento/devdocs repo](https://github.com/magento/devdocs). Fill out as much info as possible and link any GitHub issues.
 
-   In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, like `2.3.1-integration`.
+### Create a pull request
+
+Create a PR to the [magento/devdocs repo](https://github.com/magento/devdocs). Fill out as much info as possible and link any GitHub issues.
+
+In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, like `2.3.1-integration`.
+
+Choose a contribution type:
+
+- [ ] New topic - An entirely new page.
+- [ ] Content update - A substantial update or addition to an existing page; new content section with its own heading.
+- [ ] Content fix or rewrite - Small text changes or fixes: a few new sentences that clarify a subject or fixes/updates to source code or command examples.
```

Comment:
`Content fix or rewrite` is for fixing mistakes ("explicit bugs") or ambiguousness ("implicit bugs") in content.

Reply:
It can be not a text change. Fixing an incorrect diagram is also content fix or rewrite.

Reply:
> `Content fix or rewrite` is for fixing mistakes ("explicit bugs") or ambiguity ("implicit bugs") in content.

-- In this context,  it should be a radio button instead of a checklist.  Because of the GitHub template limitations, I don't think radio buttons are an option.

Reply:
See new changes.